app/js/components/cohorts/actionsComponent.jsx

```diff
@@ -125,11 +130,21 @@ class ActionsComponent extends Component {
 		};
 	}
 
+    deleteCohort(cohortId) {
+        return (e) => {
+            const apiHelper = new ApiHelper();
+            apiHelper.delete(`/cohort/${cohortId}`)
+                .then(res => {
+                    this.getAllCohorts();
```

Comment:
@andela-foladeji for most of the times, those things which you think you may need in future, you never actually get to need them. So the advice here is to assume you will never need them and use only what is for today.
That way, you do not fall in the trap of maintaining code which you may never.